runtime/lua/vim/treesitter.lua

```diff
@@ -31,9 +31,9 @@ setmetatable(M, {
 ---
 ---@param bufnr string Buffer the parser will be tied to (0 for current buffer)
 ---@param lang string Language of the parser
----@param opts table|nil Options to pass to the created language tree
+---@param opts (table|nil) Options to pass to the created language tree
 ---
----@returns table Created parser object
+---@return LanguageTree Parser object, see |LanguageTree|
```

Comment:
The EmmyLua syntax for return types is `---@return <type> [<name> [comment] | [name] #<comment>]`. Writing it like this will set `Parser` as the name of the returned variable. Although I haven't seen the notation with `#` used anywhere in the Neovim docs, maybe this can be changed in a future PR?

Reply:
I'll add the names, easy.

Reply:
Either that or just change it to `LanguageTree #Parser object, see |LanguageTree|`

Reply:
(Seems like an oversight in the specification not to allow anonymous `return` annotations with comments...)